src/Raven.Server/Documents/Indexes/Persistence/Corax/CoraxIndexingHelpers.cs

```diff
@@ -185,7 +185,15 @@ CoraxAnalyzer GetOrCreateAnalyzer(string fieldName, Type analyzerType, bool isFo
         CoraxAnalyzer CreateDefaultAnalyzer(string fieldName, Type analyzerType, bool isForQuerying)
         {
             if (analyzerType == typeof(LowerCaseKeywordAnalyzer))
+            {
+                if (indexDefinition.Version < IndexDefinitionBaseServerSide.IndexVersion.CoraxUnicodeAnalyzers_62)
+                {
+                    // BACKWARD COMPATIBILITY. Remove in v7.0
```

Comment:
No, we can't remove it. It is a valid scenario to upgrade the server from 6.0 to 7.0. We still must not break anything then.